packages/perseus/src/widgets/label-image/marker.tsx

```diff
@@ -54,8 +55,8 @@ export default class Marker extends React.Component<Props> {
         // keyboard focus is given to the marker.
         const isSelected = showSelected || focused;
 
-        let innerIcon;
-        let iconStyles;
+        let innerIcon: React.ReactElement | undefined;
+        let iconStyles: Array<CSSProperties | Falsy>;
```

Comment:
**suggestion::* You can use the `StyleType` from `@khanacademy/wonder-blocks-core` here

Reply:
thanks!